e2e/cypress/integration/at_mentions/at_mentions_spec.js

```diff
@@ -100,6 +100,7 @@ describe('at-mention', () => {
     before(() => {
         // # Login as receiver and go to "/"
         cy.apiLogin(receiver.username);
+        cy.apiSaveTeammateNameDisplayPreference('username');
```

Comment:
This has bitten us many times, might probably better to clean up state after test like in this PR - https://github.com/mattermost/mattermost-webapp/pull/5054/files/c9102c69ed2514564f1295d20969c6f989521e27#diff-55b7d7a74577283473828d0a4f8b8b12R6-R19